Restoring Span For $button['title']

ahrasis opened the pull request:
This PR is only a suggestion for easy customization from this topic:
http://www.simplemachines.org/community/index.php?topic=536271.0. The
text menu can be easily removed by css display:none with the link and
icons left intact if the span is restored. There is no other easy way to
customize removing the text from menu so far without the span. While I
am pulling this request, I will leave this totally to the SMF Team
discretion to decide.

Signed-off-by: "ahrasis" Ahmad Rasyid Ismail ahrasis@gmail.com

ahrasis committed:
Restoring Span For $button['title']

This PR is only a suggestion for easy customization from this topic:
http://www.simplemachines.org/community/index.php?topic=536271.0. The
text menu can be easily removed by css display:none with the link and
icons left intact if the span is restored. There is no other easy way to
customize removing the text from menu so far without the span. While I
am pulling this request, I will leave this totally to the SMF Team
discretion to decide.

Signed-off-by: "ahrasis" Ahmad Rasyid Ismail <ahrasis@gmail.com>

ahrasis committed:
Removing Main Menu Text In Smallest Screen

Following the early PR for adding the span in $button['title'] is
removing the text for smallest screen. As said in
http://www.simplemachines.org/community/index.php?topic=536271.msg3810677#msg3810677.
If the menu in smallest screen does look better without text, thus, this
minor css change could be useful. Hence, this PR is additional to the
first one.

Signed-off-by: "ahrasis" Ahmad Rasyid Ismail <ahrasis@gmail.com>

XinYenFon commented:
Making menu only icon in default and listing them side-by-side will disable its ability to use secondary/third menu on mobile device. That's why they are listed one-by-one in the list. Please revert changes in responsive.css rest is good to go :)

ahrasis committed:
Restoring No Span For Text Menu

Someone recently suggested that merely using css is actually possible to
remove text from menu in smaller screen in the latest post in
http://www.simplemachines.org/community/index.php?topic=536271.msg3815386#msg3815386.
I have tested his suggestion and it surprisingly works flawlessly. Since
this span is a bloat if there is no need for it to remove the text menu,
I would recommend the older code for it be restored i.e. this span be
removed again. I also modified the suggested css code in that post to be
similar to current css and remove the my previous css in this PR as
well.

Signed-off-by: "ahrasis" Ahmad Rasyid Ismail <ahrasis@gmail.com>

ahrasis committed:
Revert "Restoring No Span For Text Menu"

This reverts commit e08c29f47e6232626052d713bd8242a4d329808c.

ahrasis committed:
Restoring No Span For Main Menu Text

Someone recently suggested that merely using css is actually possible to
remove text from menu in smaller screen in the latest post in

http://www.simplemachines.org/community/index.php?topic=536271.msg3815386#msg3815386.
I have tested his suggestion and it surprisingly works flawlessly. Since
this span is a bloat if there is no need for it to remove the text menu,
I would recommend the older code for it be restored i.e. this span be
removed again. I also modified the suggested css code in that post to be
similar to current css and remove the my previous css in this PR as
well.

Signed-off-by: "ahrasis" Ahmad Rasyid Ismail <ahrasis@gmail.com>

ahrasis committed:
Revert "Restoring No Span For Main Menu Text"

This reverts commit 650c1bea0b0d4d1b5a398c85ed85aebe171ec89d.

live627 commented:
First commit was cherry picked https://github.com/SimpleMachines/SMF2.1/commit/5e6d8824877ac266883645bf73463f16b8031b11
